### PR TITLE
added debug accurracy, added cookie for drawer

### PR DESCRIPTION
**Pull request by gena**
* added accuracy for debugging
* save/restore drawer state using cookies

### CODE REVIEW

File: src/components/Drawer.vue [hunk 1]
```diff
@@ -173,6 +174,12 @@ export default {
         this.mini = true
       }
     }
+
+    window.drawer = this
```
Comment (SiggyF) [inline src/components/Drawer.vue]: please remove

File: src/components/MapComponent.vue [hunk 2]
```diff
@@ -33,13 +33,17 @@
       >
       </time-slider>
     </v-card>
+    <v-snackbar v-model="snackbarAccuracy" :timeout="8000" top right>
```
Comment (SiggyF) [inline src/components/MapComponent.vue]: move snackbar to Viewer?

File: src/utils.js [hunk 3]
```diff
@@ -30,6 +30,14 @@ export function range(start, end) {
     .map((_, idx) => start + idx);
 }
 
+/***
+ * Returns named parameter from url.
+ */
+export function getUrlParam(name) {
+  var results = new RegExp('[\\?&]' + name + '=([^&#]*)').exec(window.location.href);
```
Comment (SiggyF) [inline src/utils.js]: Or something like:
```
let params = new URLSearchParams(window.location.search)
return Object.fromEntries(params)[name]
```

File: src/components/MapComponent.vue (hunk 2 above)
Comment (gena) [reply src/components/MapComponent.vue]: Indeed, removed snackbar, now shows accuracy in infobox

File: src/utils.js (hunk 3 above)
Comment (gena) [reply src/utils.js]: yeah, simpler

**Pull request merged**